Atmosphere/style.css

```diff
@@ -0,0 +1,129 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-position: center center;
+  background-size: cover;
+  color: #fff;
+}
+
+.container {
+  padding: 16em 3em;
+  text-align: center;
+  background-image: linear-gradient(
+      to top,
+      rgba(71, 46, 104, 0.9),
+      rgba(71, 46, 104, 0.9)
+    ),
+    url(images/banner.jpg);
+}
+.container h4 {
+  font-size: 20px;
+}
+.container h1 {
+  font-size: 36px;
+}
+.btn {
+  padding: 12px 34.2px;
+  border: 1px solid #00a6d9;
+  width: 80%;
+  height: 30px;
+  background-color: #00a6d9;
+  display: inline-block;
+  text-decoration: none;
+  background-color: #00a6d9;
+  color: #ffffff;
+  text-transform: uppercase;
+}
+.container-card {
```

Comment:
Not needed 
you could have given this to the body tag
background-color: #f5f8fa;